packages/tracing-internal/src/browser/browsertracing.ts

```diff
@@ -330,6 +330,7 @@ export class BrowserTracing implements Integration {
 
       const op = 'ui.action.click';
       if (inflightInteractionTransaction) {
+        inflightInteractionTransaction._finishReason = 'interactionInterrupted';
```

Comment:
this prob needs a ts-ignore.

Actually this also might break based on how we minify, so maybe we should just add a temporary setter for this.

Reply:
Thanks for the heads up, I'll add a custom setter instead 👍